nrf_dm/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+include(${NRFXLIB_DIR}/common.cmake)
```

Comment:
Why is this included here, when it's already loaded here:
https://github.com/nrfconnect/sdk-nrfxlib/blob/9869bf438ef973c68aaeac80c3431abb83b7caaa/CMakeLists.txt#L6

```suggestion
```